examples/inference-deployments/mpt/README.md

```diff
@@ -39,3 +74,12 @@ To deploy, simply run `mcli deploy -f mpt_7b_instruct.yaml --cluster <your_clust
     ]
 }
 ```
+
+
+## Before you go
```

Comment:
would be good to include a "whats next". we can brainstorm some standard next steps to include across each example, but basic ones would be to point them to come train with us :)